docs/reference/builder.md

```diff
@@ -848,7 +848,26 @@ named `arr[0].txt`, use the following;
     ADD arr[[]0].txt /mydir/    # copy a file named "arr[0].txt" to /mydir/
 
 
-All new files and directories are created with a UID and GID of 0.
+All new files and directories are created with a UID and GID of 0, unless the
```

Comment:
How do the concepts of UID and GID 0 map to Windows?

Reply:
They do not apply/map at all.. I assume we need a note about that :)  A `Dockerfile` with the `--chown` flag on an `ADD` or `COPY` command will fail during `docker build` if it is built on a Windows Docker daemon.

This was discussed in the PR for the feature implementation itself--the other option was to "ignore" the flag on Windows but today this is **not** the way it is implemented.

Reply:
Perhaps we should add a header above this section; both for easier linking (anchor), and an introduction "This is a Linux-only feature". Also, the sections for `COPY` and `ADD` for this feature could then be combined if we think that's a good idea (_"All new files and directories that are added through `ADD` or `COPY` ...."_).  (I'm sure @mstanleyjones has better suggestions :smile:)

What is actually the owner of the files on Windows? (@johnstep?)

Reply:
@estesp What I've seen is that passing `--chown` succeeds on Windows, if a _number_ is supplied, but fails for a name since the code attempts to resolve the name by looking in `\etc\passwd` on Windows, which does not make sense.

Failing outright probably makes more sense, or at least warning, but I would prefer a more deliberate check and message. We have an issue on adding a warning, but it needs some more details: https://github.com/moby/moby/issues/35026

@thaJeztah We'll look into how the owner is set on Windows. From a quick test, it seems to be set differently based on which directory a file is copied to, but we need to verify.